python/llm/example/GPU/QLoRA-FineTuning/alpaca-qlora/README.md

```diff
@@ -1,6 +1,6 @@
-# Alpaca QLoRA Finetuning (experimental support)
+# Alpaca QLoRA & QALora Finetuning (experimental support)
 
-This example ports [Alpaca-LoRA](https://github.com/tloen/alpaca-lora/tree/main) to BigDL-LLM QLoRA on [Intel GPUs](../../README.md).
+This example ports [Alpaca-LoRA](https://github.com/tloen/alpaca-lora/tree/main) and [QALora](https://github.com/yuhuixu1993/qa-lora/) to BigDL-LLM QLoRA on [Intel GPUs](../../README.md).
```

Comment:
This example ports [Alpaca-LoRA]() to BigDL-LLM (using either [QLoRA]() or [QA-LoRA]() algorithm) on [Intel GPU].